provider/openstack/export_test.go

```diff
@@ -127,9 +129,11 @@ func GetNovaClient(e environs.Environ) *nova.Client {
 	return e.(*Environ).nova()
 }
 
-// ResolveNetwork exposes environ helper function resolveNetwork for testing
-func ResolveNetwork(e environs.Environ, networkName string, external bool) (string, error) {
-	return e.(*Environ).networking.ResolveNetwork(networkName, external)
+// ResolveNetworks exposes environ helper function resolveNetwork for testing
+func ResolveNetworkIDs(e environs.Environ, networkName string, external bool) ([]string, error) {
+	networks, err := e.(*Environ).networking.ResolveNetworks(networkName, external)
```

Comment:
Is the string to list because it might be a comma separated list, or is it because networkName might be a label that matches more than one network? (if it is the former, then we should probably change the word `networkName` to `networkNames`).

Reply:
This resolution still works on the singular name. It's just that "name" actually behaves more like a tag in OpenStack. And one name might return multiple networks.